### ISSUE TITLE
Speaking of calendars…

### ISSUE THREAD

**Issue by mscheper** [sharat87/just-a-calendar#6]
Sorry to be off topic, but this is actually about [another issue](https://github.com/psf/requests/issues/2690), for an entirely different project. But I wanted to get in touch with you, because I have an odd, perhaps morbid, curiosity about time zones.

Was it you that made the commit with the 'invalid' time zone?

If so, I'm curious whether UTC+05:18 is actually a local custom time zone, similar to [Central Western Time](https://en.wikipedia.org/wiki/UTC%2B08:45) in Australia, except that the latter exists in time zone databases, even though state and federal governments don't recognise it. ('If you don't recognise it, you can drive the 1400km and change the clocks yourself!') Maybe the real fix is to just recognise the time zone. 🤷🏼

**Comment by sharat87** [comment]
Haha, thanks for bringing that to my attention. I had no idea. The system where I made that commit is not with me anymore and I have no idea how that time zone showed up. All I remember about it was that it was a Windows system and that I was a heavy Mercurial user. I was just getting started with Git, if I recall right.

Nah it's not a real timezone here. Or anywhere I know of, for that matter. Looks like I caused more trouble than help by "fixing" that typo. 😅
